backend/inspirehep/logger/config.py

```diff
@@ -89,8 +91,27 @@ def setup_basic_logging(*args, **kwargs):
     logging.basicConfig(format="%(message)s", stream=sys.stdout, level=logging.INFO)
 
 
+def construct_failure_message(task_name, exception, affected_records):
+    """Construct a failure message based on the task name, exception, and affected records."""
+    return f"**Task name**: `{task_name}`\n **Affected record(s)**: {affected_records}\n **Error message**: {exception}"
```

Comment:
could we do something like:

Task name:

Error Message:

Affected records:
* 
* 
*